source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/StakeHolderContainer.tsx

```diff
@@ -34,75 +31,14 @@ export const StakeHolderContainer: React.FunctionComponent<IStakeHolderContainer
     }
   }, [acquisitionFile.id, getInterestHolders]);
 
-  const allInterestProperties =
-    apiInterestHolders?.flatMap(interestHolder => interestHolder.interestHolderProperties) ?? [];
-  const interestProperties = allInterestProperties
-    .filter(ip => ip.propertyInterestTypes.some(pit => pit?.id !== 'NIP'))
-    .map(ip => {
-      const filteredTypes = ip.propertyInterestTypes.filter(pit => pit?.id !== 'NIP');
-      return { ...ip, propertyInterestTypes: filteredTypes };
-    });
-  const nonInterestProperties = allInterestProperties
-    .filter(ip => ip.propertyInterestTypes.some(pit => pit?.id === 'NIP'))
-    .map(ip => {
-      const filteredTypes = ip.propertyInterestTypes.filter(pit => pit?.id === 'NIP');
-      return { ...ip, propertyInterestTypes: filteredTypes };
-    });
-
   return (
     <View
       loading={loading}
-      groupedInterestProperties={getGroupedInterestProperties(
-        interestProperties,
-        apiInterestHolders ?? [],
-        acquisitionFile,
-      )}
-      groupedNonInterestProperties={getGroupedInterestProperties(
-        nonInterestProperties,
-        apiInterestHolders ?? [],
-        acquisitionFile,
-      )}
-      legacyStakeHolders={acquisitionFile.legacyStakeholders ?? []}
+      acquisitionFile={acquisitionFile}
+      interestHolders={apiInterestHolders}
       onEdit={onEdit}
     />
   );
 };
 
-const getGroupedInterestProperties = (
```

Comment:
Is this related to this PR? If not, I'd really prefer we separate these kind of changes into their own PR.

Reply:
Thats fair. I rather keep it as is now and in the future create new PRs